GVFS/GVFS.Platform.Windows/GVFS.Platform.Windows.csproj

```diff
@@ -45,8 +45,8 @@
       <HintPath>..\..\..\packages\Microsoft.Database.Isam.1.9.4\lib\net40\Esent.Isam.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="ProjectedFSLib.Managed, Version=10.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64">
-      <HintPath>$(PackagesDir)\$(ProjFSPackage)\lib\x64\ProjectedFSLib.Managed.dll</HintPath>
+    <Reference Include="ProjectedFSLib.Managed, Version=1.0.118.0, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\..\..\packages\GVFS.ProjFS.2019.211.1-pr\lib\x64\ProjectedFSLib.Managed.dll</HintPath>

```

Comment:
@wilbaker note the hint path difference here.  The surrounding assemblies follow the "..\..\.." convention so I wanted to double check what we'd prefer here.